index.js

```diff
@@ -0,0 +1,6 @@
+const http = require('http')
+const { assert } = require('./utils')
+
+http.get('/', (res) => {
```

Comment:
We need to add a domain to this get request. See above comment about adding BASE_URL.